src/Servers/IIS/test/IIS.BackwardsCompatibility.FunctionalTests/DeployerSelector.cs

```diff
@@ -8,7 +8,8 @@ namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
     public static class DeployerSelector
     {
         public static ServerType ServerType => ServerType.IIS;
-        public static bool IsBackwardsCompatiblityTest => true;
         public static bool IsForwardsCompatibilityTest => false;
```

Comment:
Can you remove IsForwardsCompatibilityTest?

Reply:
It's used to select InProcess test project.